src/prob_causal_graph.jl

```diff
@@ -12,8 +12,7 @@ function empirical_mechanism(P, A, B)
     (; a=Z, b=P.μ[A] + Z * ( - P.μ[B]), c=P.Σ[A,A] - Z * P.Σ[B,A])
 end
 
-function prob_causal_graph(df; p=0.01, test=gausscitest)
-    @show cg = pcalg(df, p, test)
+function prob_causal_graph(df, cg; p=0.01, test=gausscitest)
```

Comment:
```suggestion
function prob_causal_graph(df, cg)
```
I think not needed anymore?